content/en/docs/tasks/administer-cluster/migrating-from-dockershim/topics-on-dockershim-and-cri-compatible-runtimes.md

```diff
@@ -0,0 +1,27 @@
+---
+reviewers:
+- tbd
+title: Topics on dockershim and CRI-compatible runtimes
+content_type: concept
```

Comment:
We usually don't put concept pages in the tasks section.

Does this belong inside https://k8s.io/docs/reference/ perhaps?

Reply:
Thanks, @sftim. I don't really understand the k8s docs' paradigm for organizing content by content type. Maybe we could huddle on this briefly in a day or so? 

I agree that this "list-of-links" topics seems more like a reference topic, so I've updated this line to:  
```suggestion
content_type: reference
```
Where should it go within the `docs/reference` folder?

Reply:
Thank you, @sftim. PTAL and let me know what additional changes might be needed.